site/content/contribute/server/developer-setup/arch.md

```diff
@@ -57,13 +57,13 @@
 8. Clone the Mattermost source code from your fork:
 
     ```sh
-    git clone https://github.com/YOUR_GITHUB_USERNAME/mattermost-server.git
+    git clone https://github.com/YOUR_GITHUB_USERNAME/mattermost-server.git $(go env GOPATH)/src/github.com/mattermost/mattermost-server
```

Comment:
Since go 1.12 you don't need to put your code into `GOPATH`. Can you elaborate why this might be needed?